packages/tbas/src/types/glogal.d.ts

```diff
@@ -0,0 +1,5 @@
+declare global {
+  interface Window {
+    ethereum: any
```

Comment:
we could use a real type here I think

Reply:
Stackoverflow suggestion. We won't use it tho as it's a server function. Deleted